### DISCUSSION TITLE
Does this library work offline?

### DISCUSSION THREAD

**Discussion by ingeniousgenius** [supabase/supabase#827] [Questions] [answered]
Hey there!
Just found out about this project and it looks great. I'm looking to build a completely offline-first application and was wondering how compatible this would be with that type of architecture?

**Comment by OrhanTozan** [comment]
Off-topic: I think this Discussion can be but in the Q&A Category :)

**Comment by MildTomato** [comment, marked as answer]
Feature request discussion for this can be found [here](https://github.com/supabase/supabase/discussions/357)
